src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+.formBig {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.inputBig {
+  height: 70px;
+  width: 100%;
+  left: 0;
+  top: 0;
+  border-radius: 4px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.picture {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.inputBig:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.inputBig:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.inputBig::placeholder {
+  font-family: Avenir, sans-serif;
+}
+
+.formSmall {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.inputSmall {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  height: 42px;
+  width: 100%;
+  left: 0;
+  top: 0;
+  padding-left: 33px;
+}
```

Comment:
check out point 2 in the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)